### PR TITLE
DE-6509: Kafka-Iceberg blog code

### PR DESCRIPTION
**Pull request by rmoff**
- **Build Kafka-Iceberg demo with Decodable**
- **Add ST config**
- **Add README**

### CODE REVIEW

File: kafka-iceberg/decodable/README.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+```
+
+## Create Iceberg connection
+
+You'll need to put your own database, region, and role-arn in here.
+
+```bash
+decodable connection create                                                        \
```
Comment (gunnarmorling) [inline kafka-iceberg/decodable/README.md:150]: Can we use declarative instead? It'll also avoid the need for the yq magic below (albeit that's cool, didn't occur to me so far!).

**Review by gunnarmorling — APPROVED** [review]
Nice one; one suggestion inline.

File: kafka-iceberg/decodable/README.md (hunk 1 above)
Comment (rmoff) [reply kafka-iceberg/decodable/README.md:150]: good idea. 
next time :)

**Comment by gunnarmorling** [comment]
Why not this time? Declarative should be clearly better for this sort of stuff, so IMO we should use it for all new examples ideally from now on.

**Comment by rmoff** [comment]
because this blog has been sitting at 90% for weeks and I just need to ship the damn thing :)

**Comment by gunnarmorling** [comment]
But then you could also spend that extra half day for making it perfect. Anyways, it's ok, that way we can make a separate post on Declarative, without stealing its thunder.

**Pull request merged**